extensions/extensions-security/extensions-authzforce/src/main/java/com/boozallen/aissemble/security/authorization/policy/AissembleAttributeProvider.java

```diff
@@ -156,22 +145,22 @@ public <AV extends AttributeValue> AttributeBag<AV> get(AttributeFqn attributeGU
 
 
     /**
-     * Returns a request for a simple, AIOps attribute by attribute id and subject.
+     * Returns a request for a simple, aissemble attribute by attribute id and subject.
      *
      * @param id
      *            attribute id
      * @param subject
      *            subject name
      * @return collection of attributes or null if none found
      */
-    public Collection<com.boozallen.aiops.security.authorization.policy.AttributeValue<?>> getAiopsAttributeByIdAndSubject(String id,
-            String subject) {
+    public Collection<com.boozallen.aissemble.security.authorization.policy.AttributeValue<?>> getAissembleAttributeByIdAndSubject(String id,
+                                                                                                                               String subject) {
```

Comment:
A: looks like extra spaces added here

Reply:
I think GH is making the formatting look a bit off, in the IDE it looks more pleasing.